tests/test_wave_01.py

```diff
@@ -182,13 +183,14 @@ def test_moves_movie_from_watchlist_to_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
+    assert movie_to_watch in updated_data["watched"]
```

Comment:
👍 Here, it would also be a good idea to check that the movie we _didn't_ expect to move is still in the watchlist, and the other movie that was already in the watched list is still there.